### ISSUE TITLE
can't get installation of hologram completed

### ISSUE THREAD

**Issue by dbyy** [MarkSlat/OctoPrint-Hologram#10]
Hello, 
I tried now several times to install this plugin and every time it seems to hang in the phase "preparing meta data (pyproject.toml)"

![Octoprint hologram 2024-11-27T11-25-04](https://github.com/user-attachments/assets/1071a86b-70da-44ff-b096-d9194e8a2320)

How can I fix the problem so that I can use the plugin?

And btw. "libopenblas-base" is install on my pi.

And response is much appreciated

I'm running Octoprint v 1.10.c octopi v 1.0

**Comment by MarkSlat** [comment]
Hello,
What raspberry pi are you using?
When I installed it on a raspberry pi 3 it took me about an hour from what I can remember and only a few min on a raspberry pi 4.

**Comment by dbyy** [comment]
> Hello, What raspberry pi are you using? When I installed it on a raspberry pi 3 it took me about an hour from what I can remember and only a few min on a raspberry pi 4.

It is a Pi 3-B rev 1.2.
Maybe that's the reason. 
I just must give it more time, but its awful long.
Thanks

**Comment by dbyy** [comment]
Switch to a rPi 4 and the installation went much faster.
Case closed. Thanks Mark

**Comment by dbyy** [comment]
A final comment/question:
I can't figure it out how to access (display) the hologram.
I checked every page of octoprint but can't find the button.
Any suggestion?

**Comment by dbyy** [comment]
> A final comment/question: I can't figure it out how to access (display) the hologram. I checked every page of octoprint but can't find the button. Any suggestion?

How can I access the plug-in octoprint? Anybody knows and are to answer?

**Comment by MarkSlat** [comment]
That's weird it should be on the main page under a tab. Did you have any issues locking in the build plate?

**Comment by dbyy** [comment]
> MarkSlat ...locking in the build plate ??
Is that part of the plug-in?

**Comment by MarkSlat** [comment]
Yes in settings there is a section for the hologram plugin and there you have to line up a box with your build plate. From there you can go to the hologram tab and select a part to render.
